app/src/main/java/com/grignon/shanghai/entity/Room.java

```diff
@@ -0,0 +1,77 @@
+package com.grignon.shanghai.entity;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Room {
+    private String name;
+    private String code;
+    private Integer type;
+    private Player admin;
+    private List<Player> players;
```

Comment:
Tous tes paramètres peuvent être final ici aussi 
```suggestion
    private final String name;
    private final String code;
    private final Integer type;
    private final Player admin;
    private final List<Player> players;
```